src/main/resources/static/js/datastream.js

```diff
@@ -125,158 +125,158 @@ function createDS() {
                 )
             )
-            .append($('<td>')
-                .append($('<input>')
-                    .attr('size', 15)
-                    .attr('type', 'text')
+            .append($("<td>")
+                .append($("<input>")
+                    .attr("size", 15)
+                    .attr("type", "text")
                 )
             )
-            .append($('<td>')
-                .append($('<input>')
-                    .attr('size', 15)
-                    .attr('type', 'text')
+            .append($("<td>")
+                .append($("<input>")
+                    .attr("size", 15)
+                    .attr("type", "text")
                 )
             )
-            .append($('<td>')
-                .append($('<button>')
-                    .attr('class', 'btn btn-secondary')
-                    .attr('onclick', 'delRow($(this))')
-                    .attr('style', 'width: auto;')
-                    .html('<span class="fas fa-minus" ></span>')
+            .append($("<td>")
+                .append($("<button>")
+                    .attr("class", "btn btn-secondary")
+                    .attr("onclick", "delRow($(this))")
+                    .attr("style", "width: auto;")
+                    .html("<span class='fas fa-minus'></span>")
                 )
             )
         );
 
-    $('.selectProperties').last().select2({
+    $(".selectProperties").last().select2({
         data: streamProperties,
-        placeholder: 'Choose a Property',
-        width: 'style',
+        placeholder: "Choose a Property",
+        width: "style",
         dropdownAutoWidth: true
-    }).trigger('change').on('select2:select', function (e) {
-        $(this).find('option:selected').attr('data-value', e.params.data['data-value']);
+    }).trigger("change").on("select2:select", function (e) {
+        $(this).find("option:selected").attr("data-value", e.params.data["data-value"]);
     });
 
-    $('.selectStreamType').last().select2({
+    $(".selectStreamType").last().select2({
         data: streamTypes,
-        placeholder: 'Choose a datatype',
-        width: 'style',
+        placeholder: "Choose a datatype",
+        width: "style",
         dropdownAutoWidth: true
-    }).trigger('change').on('select2:select', function (e) {
-        $(this).find('option:selected').attr('data-value', e.params.data['data-value']);
+    }).trigger("change").on("select2:select", function (e) {
+        $(this).find("option:selected").attr("data-value", e.params.data["data-value"]);
     });
 }
 
 function showSensorModal(url) {
-    var $modal = $('#dsfooter').find('button:eq(0)');
-    $modal.attr('onclick', 'createSensor()');
-    $modal.html('Create Sensor');
-    modal('dsdialog', url, initSensor, 'Create a Sensor');
+    var $modal = $("#dsfooter").find("button:eq(0)");
+    $modal.attr("onclick", "createSensor()");
+    $modal.html("Create Sensor");
+    modal("dsdialog", url, initSensor, "Create a Sensor");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['initSensor' is not defined. (no-undef)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)